db/migrate/20220729165329_create_gardens.rb

```diff
@@ -2,8 +2,10 @@ class CreateGardens < ActiveRecord::Migration[5.2]
   def change
     create_table :gardens do |t|
       t.string :name
-      t.string :cardinal_direction
+      t.integer :cardinal_direction
       t.string :notes
```

Comment:
will this need to be a text block option instead of a string? Not sure how that works in postgres